libraries/AP_HAL_ChibiOS/hwdef/CORVON743V1/hwdef.dat

```diff
@@ -0,0 +1,168 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the CORVON743V1 hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID. See Tools/AP_Bootloader/board_types.txt
+APJ_BOARD_ID AP_HW_CORVON743V1
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 12
+define CH_CFG_ST_RESOLUTION 16
+
+# flash size
+FLASH_SIZE_KB 2048
+FLASH_RESERVE_START_KB 128
+
+define HAL_STORAGE_SIZE 32768
+STORAGE_FLASH_PAGE 14
+
+# default to all pins low to avoid ESD issues
+DEFAULTGPIO OUTPUT LOW PULLDOWN
+
+# order of UARTs (and USB)
+SERIAL_ORDER  OTG1 USART1 USART2 USART3 UART4 USART6 UART7 UART8
```

Comment:
If you wanted you could put "EMPTY" between 4 and 6 and then the serial numbers would match the uart numbers